lib/capybara_webkit_builder.rb

```diff
@@ -89,4 +89,17 @@ def build_all
     build &&
     clean
   end
+
+  def make(target = "")
+    env_hide("CDPATH") { sh("#{make_bin} #{target}") }
+  end
+
+  def env_hide(name)
+    @stored_env ||= {}
+    @stored_env[name] = ENV.delete(name)
+
+    yield
+
+    ENV[name] = @stored_env[name]
```

Comment:
Do we have to do this at all? It won't affect the parent shell, and we don't ever want `CDPATH` around in builds, right?

If you prefer to isolate the changes, should you add an `ensure` here in case the `yield` fails?